src/proposals/proposalTypes/CrossChainProposal.sol

```diff
@@ -163,43 +160,43 @@ abstract contract CrossChainProposal is
     /// @notice get the calldata for the Artemis Governor to propose the cross chain proposal on Moonbeam
     /// @param temporalGovernor address of the cross chain governor executing the calls
     /// @param wormholeCore address of the wormhole core contract
-    function getArtemisGovernorCalldata(
+    function getMultichainGovernorCalldata(
         address temporalGovernor,
         address wormholeCore
     ) public returns (bytes memory) {
         require(
             temporalGovernor != address(0),
-            "getArtemisGovernorCalldata: Invalid temporal governor"
+            "getMultichainGovernorCalldata: Invalid temporal governor"
         );
         require(
             wormholeCore != address(0),
-            "getArtemisGovernorCalldata: Invalid womrholecore"
+            "getMultichainGovernorCalldata: Invalid womrholecore"
         );
 
         bytes memory temporalGovCalldata = getTemporalGovCalldata(
             temporalGovernor
         );
 
         return
-            getArtemisGovernorCalldata(
+            getMultichainGovernorCalldata(
                 temporalGovernor,
                 wormholeCore,
                 temporalGovCalldata
             );
     }
 
-    function getArtemisGovernorCalldata(
+    function getMultichainGovernorCalldata(
```

Comment:
this method looks equal to the one we have on HybridProposal. Wouldn't it be better to make HybridProposal inherit from CrossChainProposal?